CommonTools/RecoAlgos/python/HBHENoiseFilter_cfi.py

```diff
@@ -1,25 +1,24 @@
+#
+# This is a replacement for the original HBHENoiseFilter configuration.
+# See https://twiki.cern.ch/twiki/bin/viewauth/CMS/HCALNoiseFilterRecipe.
+# Note that this replacement relies on having the HcalNoiseSummary in the
+# event record but not necessarily HBHENoiseFilterResult.
+#
 import FWCore.ParameterSet.Config as cms
 
+# Module which will remake HBHENoiseFilterResult
+from CommonTools.RecoAlgos.HBHENoiseFilterResultProducer_cfi import HBHENoiseFilterResultProducer
+MakeHBHENoiseFilterResult = HBHENoiseFilterResultProducer.clone()
```

Comment:
I'm missing the point: why is this producer clone needed if one is already available and running with the same configuration?
Just use the already available HBHENoiseFilterResultProducer instance.